app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Sidebar/Wishlist.php

```diff
@@ -109,6 +109,19 @@ public function getIdentifierId($item)
     {
         return $item->getId();
     }
+    
+    /**
+     * Retrieve display block availability
+     *
+     * @return bool
```

Comment:
Not sure if this is better:
```suggestion
     * @inheritDoc
 ```